examples/react-todos-app/project.json

```diff
@@ -42,8 +42,7 @@
       "outputs": ["{options.reportsDirectory}"],
       "options": {
         "config": "examples/react-todos-app/vite.config.ts",
-        "reportsDirectory": "../../coverage/examples/react-todos-app",
-        "coverage": true
+        "reportsDirectory": "../../coverage/react-todos-app"
```

Comment:
It's because of the `coverage-plugin`, specifically `getNxCoveragePaths` helper expects it in `project.json`. I plan to create an issue for `getNxCoveragePaths` problems (also doesn't work with Jest, BTW). But for now this workaround is needed. Or we remove the helper from `code-pushup.config.ts` for now, that's fine with me also.

Reply:
Created #536